cinema/models.py

```diff
@@ -95,7 +101,7 @@ def validate_ticket(row, seat, cinema_hall, error_to_raise):
                 raise error_to_raise(
                     {
                         ticket_attr_name: f"{ticket_attr_name} number "
-                        f"must be in available range: "
+                        "must be in available range: "
                         f"(1, {cinema_hall_attr_name}): "
                         f"(1, {count_attrs})"
```

Comment:
The string concatenation across multiple lines could be simplified using an f-string for better readability. Consider using a single f-string to format the entire error message.